docker/release/Dockerfile.distroless

```diff
@@ -95,5 +95,8 @@ LABEL install="podman volume create keylime-agent"
 LABEL uninstall="podman volume rm keylime-agent"
 LABEL run="podman run --read-only --name keylime-agent --rm --device /dev/tpm0 --device /dev/tpmrm0 -v keylime-agent:/var/lib/keylime -v /etc/keylime:/etc/keylime:ro --tmpfs /var/lib/keylime/secure:rw,size=1m,mode=0700 -dt IMAGE"
 
+# Create a system user 'keylime' to allow dropping privileges
+RUN useradd -s /sbin/nologin -r -G tss keylime
```

Comment:
Why not to use USER?

https://docs.docker.com/engine/reference/builder/#user

Reply:
AFAIK, `USER` only sets which user will run the container process, it does not create an user.

And we need to run the container as root to access the files that need privilege, such as the `/sys/kernel/security/tpm0/binary_bios_measurements` or the TPM device.

If you know another way, please let me know